relay-dynamic-config/src/defaults.rs

```diff
@@ -440,6 +440,9 @@ fn span_metrics(transaction_extraction_enabled: bool) -> impl IntoIterator<Item
                 Tag::with_key("span.group")
                     .from_field("span.sentry_tags.group")
                     .when(know_modules_condition.clone() | app_start_condition.clone()),
+                Tag::with_key("span.category")
```

Comment:
@Zylphrex says this should be on `know_modules_condition` - but I need it for AI at least

Reply:
Does this mean that `span.category` should be extracted for all known module conditions, besides AI spans? The tag is also extracted for a bunch of HTTP and DB spans.

Reply:
I suppose so, but I'll just make it AI only for now until someone else wants it